ncm-afsclt/src/main/perl/afsclt.pm

```diff
@@ -801,197 +368,113 @@ sub update_afs_cells ( $$ ) {
 }
 
 ##########################################################################
-sub Unconfigure_Cell {
+sub Configure_Afsd_Args {
 ##########################################################################
-  unless ($NoAction) {
-   my ($self,$config)=@_;
-
-   return unless authconfig_OK($self,$authconfig);
-
-  $self->warn("Unconfigure_Cell does not work for now (authconfig broken)");
-  return;
+  my ($self,$config)=@_;
 
-   my $comm=$authconfig . " --nostart --kickstart --disableafs";
-   my $s=`$comm`;
-    if ($? >> 8) {
-      $self->error("can't run $s, no changes made");
-      return;
-    }
-    $self->OK("Unconfigured AFS client");
+  my $path = "$mypath/afsd_args";
+  if ($config->elementExists($path)) {
+      if (! $NoAction) {
+	  my $args = $config->getElement($path)->getTree();
+	  my $content;
+	  $content .= "$_:" . $args->{$_} . "\n" for sort keys %$args;
```

Comment:
why no `print $fh "$_:" . $args->{$_} . "\n" for sort keys %$args;`, the extra variable makes no sense